### PR TITLE
Cosmwasm std 2 update

### PR DESCRIPTION
**Pull request by Kayanski**
This PR aims at updating cosmwas-std to 2.0 version.

I am blocked because Osmosis-std and test-tube need cosmwasm-std 1. 

- Use cw-multi-test branch for dev, don't rush it : `update/add-ibc-support-cosmwasm2`
- Use clone-cw-multi-test branch for dev, don't rush it : `update/local-support-cosmwasm2`

Gotta wait until osmosis test tube is separated OR osmosis-std updates

Temporary Ignored : 
- cw-orch-compatibility test
- Clone Testing (because clone-cw-multi-test needs cw-orch to be updated first)
- Osmosis Test tube (because osmosis-std hasn't updated)

### CODE REVIEW

**Comment by Buckram123** [comment]
Closes ORC-151

File: cw-orch-interchain/examples/starship_ica_demo.rs [hunk 1]
```diff
@@ -13,13 +10,13 @@ pub const STARGAZE: &str = "stargaze-1";
 pub const JUNO_FUNDS_DENOM: &str = "ujuno";
 
 fn main() {
```
Comment (CyberHoward) [inline cw-orch-interchain/examples/starship_ica_demo.rs]: Have you ensures this test works? Should we add this test in CI?

Comment (Buckram123) [reply cw-orch-interchain/examples/starship_ica_demo.rs]: Yep, just checked, it runs. Do you think we want starship tests in CI?

Comment (CyberHoward) [reply cw-orch-interchain/examples/starship_ica_demo.rs]: If possible yes, but it could be pain!

Comment (CyberHoward) [reply cw-orch-interchain/examples/starship_ica_demo.rs]: i think the starship docs might have some snippet for it

**Review by CyberHoward — APPROVED** [review]

**Pull request merged**